tecken/base/symboldownloader.py

```diff
@@ -108,7 +108,10 @@ def exists_in_source(source, key):
 @metrics.timer_decorator("symboldownloader_public_exists")
 def check_url_head(url):
     session = session_with_retries()
-    return session.head(url).status_code == 200
+    resp = session.head(url)
+    if resp.status_code not in (200, 404):
+        logger.error(f"check_url_head: {url} status code is {resp.status_code}")
+    return resp.status_code == 200
```

Comment:
This was a bug and made debugging with minio really hard because minio would return a 403 which is not 200. Previously, a 403 would just cause this to return False, but 403 is an error scenario (500, too) and we should log an error.